### PR TITLE
Feat/approve tokens changes

### PR DESCRIPTION
**Pull request by Faouzijedidi1**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

**Tests performed by the developer**:

**Tips for QA testing**:

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def build_extensions(self):
 
 def main():
     cpu_count = os.cpu_count() or 8
-    version = "20221128"
+    version = "20221221"
```
Comment (cardosofede) [inline setup.py:35]: why the version is different?

**Review by cardosofede — CHANGES_REQUESTED** [review]
check those observations!

**Review by rapcmia — APPROVED** [review]
LGTM 

- Installation source/docker, ok 
- Setup simple pancakeswap-bsc on source 
- Test with tokens that needs to be approved
	- Confirmed tokens needs to be enabled on connector
	- Run `gateway approve-token [connector] [token]`
		- Displayed `Token XXXX approval tx is pending`
		- Displayed `Token XXXX is approved for spending for xxxx for wallet xxxx` on both log and output panel 
	- Successfully approved and amm-arb worked ok 
- Test docker with pangolin-avalanche
	- same test done with source, all ok 
- Displayed help on `gateway approve-token -h`
- Test invalid tokens, getting expected error for unsupported tokens 
- Execute `gateway approve-token` while AMM-arb is running, successfully reproduce `Tokens [token] is not approved for spending. Please use manual approval.`

**Review by cardosofede — APPROVED** [review]
LGTM!

**Pull request merged**